lib/utils/datasetState.cpp

```diff
@@ -36,7 +36,6 @@ std::ostream& operator<<(std::ostream& out, const datasetState& dt) {
     return out;
 }
 
-// TODO: this is a very weird place for this routine to be. Put it somewhere more sane.
```

Comment:
It's still in a weird place. Did you just move this into an issue?

Reply:
Where else should it be? I thought that now that it's all in `datasetState.*` we are good, because that's the only place where it is used.

Reply:
https://github.com/kotekan/kotekan/issues/601